app/models/template_input.rb

```diff
@@ -71,7 +71,7 @@ def prevent_delete_if_template_is_locked
   end
 
   def check_if_template_is_locked
-    if template&.locked
+    if template&.locked && !ForemanSeeder.is_seeding
```

Comment:
What about depending on the template definition of it directly?
```suggestion
    if template&.run_temlate_changes_validation?
```

Reply:
aside from the typo, yes! ;)

Reply:
:facepalm: yeah rather using the correct method name :laughing:

Reply:
The method is private :(